Source/Core/Matrix3.js

```diff
@@ -72,13 +74,8 @@ define([
      */
     Matrix3.pack = function(value, array, startingIndex) {
         //>>includeStart('debug', pragmas.debug);
-        if (!defined(value)) {
-            throw new DeveloperError('value is required');
-        }
-
-        if (!defined(array)) {
-            throw new DeveloperError('array is required');
-        }
+        Check.typeOf.object(value, 'value');
+        Check.typeOf.object(array, 'array');
```

Comment:
This would be `Check.defined(array, 'array');`

Also for all other arguments below that are of type arrays.